plenary/plenary-35.yaml

```diff
@@ -0,0 +1,325 @@
+metadata:
+  title: Resolutions of the 35th plenary meeting of ISO/TC 154 at Berlin
+  date: 2016-10-21
+  source: ISO/TC 154 Secretariat
+resolutions:
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Adoption of a PWI for the new part ISO 14533-4"
+    identifier: 2016-01
+    considerations:
+      - type: considering
+        date_effective: 2016-10-21
+        message: considering the SK/SOSMT proposal report status for the new part ISO 14533-4
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2016-10-21
+        message: resolves to adopt a PWI as the new part ISO 14533-4 on “Attributes pointing to external Proof of Existence objects used in Long term signature formats (PoEAttributes)”, and requests SK/SOSMT to prepare the final working draft for NWIP ballot
+
+
+  - category:
+    dates:
+      - 2016-10-21
```

Comment:
No category is shown in the original document file. Is it ok to leave it in blank?

Reply:
Yes, I think then we can remove the `category:` line.

Reply:
Ok, changes applied.

Reply:
Thanks!